test/tstQueryTreeTraversalPolicy.cpp

```diff
@@ -0,0 +1,177 @@
+/****************************************************************************
+ * Copyright (c) 2012-2020 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "ArborX_EnableDeviceTypes.hpp" // ARBORX_DEVICE_TYPES
+#include <ArborX_LinearBVH.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+#include <algorithm>
+#include <iostream>
+#include <random>
+#include <tuple>
+
+#include "Search_UnitTestHelpers.hpp"
+
+BOOST_AUTO_TEST_SUITE(TraversalPolicy)
+
+namespace tt = boost::test_tools;
+
+BOOST_AUTO_TEST_CASE_TEMPLATE(buffer_optimization, DeviceType,
+                              ARBORX_DEVICE_TYPES)
+{
+  using ExecutionSpace = typename DeviceType::execution_space;
+
+  auto const bvh = make<ArborX::BVH<typename DeviceType::memory_space>>(
```

Comment:
Fix.

Reply:
What is there to fix?

Reply:
Ah, my bad, I did not realize that this was the test that only worked with `BVH`. Still jarring to see this instead of `make<Tree>`, but ok.